### DISCUSSION TITLE
Does Comment.nvim support Vue?

### DISCUSSION THREAD

**Discussion by patrickphat** [numToStr/Comment.nvim#259] [Q&A] [answered]
For me it seems like no Vue supported at this time because both <script> and <template> section are commented the same way where they should be different.
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/45748186/194840509-6faa02a9-e2b6-4af9-8b9a-a92132196a8f.png">

**Comment by numToStr** [comment, marked as answer]
For me it is working fine. Make sure that you have `tree-sitter-vue` parser installed.

**Reply by patrickphat** [reply]
Thanks @numToStr! For those who need:
```
yarn install global tree-sitter-vue tree-sitter
```

**Reply by numToStr** [reply]
@patrickphat This is not right way to install any parser in neovim. Use `nvim-treesitter` plugin instead.
